components/aws/sagemaker/tests/integration_tests/resources/definition/training_pipeline.py

```diff
@@ -54,7 +53,6 @@ def training_pipeline(
         checkpoint_config=checkpoint_config,
         debug_hook_config=debug_hook_config,
```

Comment:
debug_hook_config and debug_rule_config could be removed as well now ?

Reply:
What is the benefit of removing them?